sklearn/sklearn-template/template/trainer/model.py

```diff
@@ -20,34 +20,48 @@
 from sklearn import pipeline
 from sklearn import preprocessing
 
+import numpy as np
+
 from trainer import metadata
 
 
 def get_estimator(flags):
-  classifier = ensemble.RandomForestClassifier()
-
-  # TODO(cezequiel): Make use of flags for hparams
-  _ = flags
+  # TODO: Allow pre-processing to be configurable through flags
+  classifier = ensemble.RandomForestClassifier(**flags)
 
   numeric_transformer = pipeline.Pipeline([
-      ('imputer', impute.SimpleImputer(strategy='median')),
-      ('scaler', preprocessing.StandardScaler()),
+    ('imputer', impute.SimpleImputer(strategy='median')),
```

Comment:
Everywhere:
Indents are 4 spaces for continuing lines (see internal Python style guide).
For OSS Python style, I believe we should be using 4-space indents for all: http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

However, since our dev environments are configured for 2-space indents by default, would you consider going with 2-space indent (4-space continuation) instead?

Reply:
Agree, I have re-formatting to 2-space indent + 4-space continuation